Ryujinx/Ui/MainWindow.cs

```diff
@@ -865,5 +777,10 @@ private static int FileSizeSort(ITreeModel model, TreeIter a, TreeIter b)
                 return 0;
             }
         }
+
+        public static void SaveConfig()
+        {
+            ConfigurationState.Instance.ToFileFormat().SaveConfig(System.IO.Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "Config.json"));
```

Comment:
```suggestion
            ConfigurationState.Instance.ToFileFormat().SaveConfig(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "Config.json"));
```
Since there already is a using

Reply:
Conflicting with GTK.Widget.IO